api/pom.xml

```diff
@@ -48,6 +48,12 @@
 
         <dependency>
             <groupId>org.motechproject.nms</groupId>
+            <artifactId>props</artifactId>

```

Comment:
Based on this morning's conversation, should we change the name to config or configuration?